bangank36/shopify-order-status-editor
# 🛍️ Shopify Order Status: kick off

## Summary
- This PR update the dependencies to overcome the SSL issue when running the demo
- Update the credit and project name

## Testing
- Open the index page on localhost
- Confirm everything is working as before

## Reference
#1
